ComputeClient/Compute.Contracts/Requests/Snapshot/SnapshotListOptions.cs

```diff
@@ -47,6 +47,11 @@ public class SnapshotListOptions : FilterableRequest
 		/// </summary>
 		public const string DescriptionField = "description";
 
+		/// <summary>
```

Comment:
Changes seem fine. Think we a simple test in _ServerAccessorTests.ListServersWithFilterOptions_ReturnsResponse()_ which was used to verify the filter is setup is correct.

We could add probably something similar to _SnapshotAccessorTests_ .